docs/dev_tools/third_party.md

```diff
@@ -93,6 +94,23 @@ Please download and install it.
 	
 Make and Eclipse exports on POSIX-like operating systems targeting Nordic devices require the `srec_cat` executable from the [sRecord](http://srecord.sourceforge.net) package. It may be available from your package manager (such as apt-get or Brew). 
 
+### GNU ARM Eclipse (managed CDT projects) with GNU ARM Embedded Toolchain
+
+The [GNU ARM Eclipse](http://gnuarmeclipse.github.io) exporter generates ready to run managed CDT projects, and strives to do it as accurately as possible, considering the large number of explicit configuration options GNU ARM Eclipse uses. 
+
```

Comment:
Could we change this to "The [GNU ARM Eclipse](http://gnuarmeclipse.github.io) exporter generates ready to run managed CDT projects for a large number of explicit configuration options." to shorten it and get rid of the personification?